modules/gcp/src/main/scala/com.snowplowanalytics.snowplow.lakes/GcpApp.scala

```diff
@@ -19,5 +21,15 @@ object GcpApp extends LoaderApp[PubsubSourceConfig, PubsubSinkConfig](BuildInfo)
 
   override def badSink: SinkProvider = PubsubSink.resource(_)
 
-  override def isDestinationSetupError: DestinationSetupErrorCheck = TableFormatSetupError.check
+  override def isDestinationSetupError: DestinationSetupErrorCheck = {
+    // Destination bucket doesn't exist
+    case e: GoogleJsonResponseException if e.getDetails.getCode == 404 =>
```

Comment:
I'm a big fan of cats triple equals in places like this

```scala
  case e: GoogleJsonResponseException if e.getDetails.getCode === 404 =>
```

Reply:
I updated !